### ISSUE TITLE
In Progress?

### ISSUE THREAD

**Issue by RayWillett** [LiamHughes599/PencilKata#3]
Are we still working on this? It's been a bit since the last commit.

@LiamHughes599

**Comment by LiamHughes599** [comment]
Yes, I still plan to finish this. I took a break from it. I was feeling a bit burnt on it tbh. I'm sorry, I should've updated you a bit more. But yea i restructured the app.js page. I'm still a bit stuck on going about the editing functionality. I have fullPrint as an array of strings that outputs with a space in between the strings. I was hoping to keep it as an array for the counting letters function and to push the input to fullPrint functions. And once I realized it, I took a break to enjoy the summer. I'm back at it again though, classes start tomorrow so I'm getting back to work. The good news is that while working with all of this I do have a much better understanding of JavaScript. There are things I would've done different at the start, like not wasting a month on this one nodejs tutorial that wasn't actually very useful lol. But its all part of learning lol. But yes, I've committed the progress that I have now.
